.github/workflows/validate-bicep.yaml

```diff
@@ -52,27 +52,21 @@ jobs:
           bicep --version
       - name: Check out repo
         uses: actions/checkout@v4
-      - name: Create a secure local registry
-        id: create-local-registry
-        uses: ./.github/actions/create-local-registry
-        with:
-          secure: "true"
-          registry-name: ${{ env.LOCAL_REGISTRY_NAME }}
-          registry-server: ${{ env.LOCAL_REGISTRY_SERVER }}
-          registry-port: ${{ env.LOCAL_REGISTRY_PORT }}
+      - name: Create a temporary file system 
+        run:
+          mkdir tmp
```

Comment:
suggestion - can give give this a more meaningful name for future debugging? eg: `mkdir tmp-radius-bicep-provider` or similar